README.md

```diff
@@ -323,6 +366,38 @@ allprojects {
 }
 ```
 
+7. Additional steps for using MLKit for text/face/barcode recognition
```

Comment:
I've noticed that qr code recognition works for me on android without following instructions for integration of firebase